docs/config.md

```diff
@@ -769,6 +769,27 @@ command = ["head", "-n", "10"]
 patterns = ["numbers.txt"]
 ```
 
+### Disabling and enabling tools
+
+Tools can be disabled and enabled with the optional `enabled` config. This
+allows you to define tools globally but enable them only for specific
+repositories.
+
+In the user configuration, define a disabled tool for running rustfmt:
+
+```toml
+[fix.tools.rustfmt]
+enabled = false
+command = ["rustfmt", "--emit", "stdout"]
+patterns = ["glob:'**/*.rs'"]
+```
```

Comment:
nit: can you also update `config-schema.json` and doc string of `FixArgs`?

Reply:
Whoops, I didn't realize that's what config-schema.json was for (it's obvious in retrospect). Updated.